src/propagation.rs

```diff
@@ -1,10 +1,16 @@
+// Originally retired from davidB/tracing-opentelemetry-instrumentation-sdk
```

Comment:
Not his file.

This code I wrote myself 😃

Reply:
updated